test/mock-data.js

```diff
@@ -1,2 +1,4 @@
-export const singleMatrix = require('./single-matrix.json');
-export const multipleMatrices = require('./multiple-matrix.json');
+import singleMatrix from './single-matrix.json';
+import multipleMatrices from './multiple-matrix.json';
+
+export {singleMatrix, multipleMatrices};
```

Comment:
It looks like the full Import Assertions (https://github.com/tc39/proposal-import-assertions) syntax will be required in Node.js (https://nodejs.org/api/esm.html#json-modules).  While that is all experimental, I'd recommend using a current syntax that doesn't require command flags to load JSON.  Something like the following:

```suggestion
import {createRequire} from 'module';
const requireJson = createRequire(import.meta.url);

export const singleMatrix = requireJson('./single-matrix.json');
export const multipleMatrices = requireJson('./multiple-matrix.json');
```